src/test/java/io/specto/hoverfly/junit/dsl/SingleQuoteHttpBodyConverterTest.java

```diff
@@ -0,0 +1,27 @@
+package io.specto.hoverfly.junit.dsl;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SingleQuoteHttpBodyConverterTest {
+
+    @Test
+    public void shouldConvertSingleQuotes() {
+        final SingleQuoteHttpBodyConverter singleQuoteHttpBodyConverter =
+            SingleQuoteHttpBodyConverter.jsonWithSingleQuotes("{'bookingId':'1'}");
+
+        assertThat(singleQuoteHttpBodyConverter.body())
+            .isEqualTo("{\"bookingId\":\"1\"}");
+    }
+
+    @Test
```

Comment:
In current implementation, scape char is `\\` so `"{'merchantName':'McDonals\\'s'}"` is transformed to `"{\"merchantName\":\"McDonals's\"}"`